tools/pipelines/templates/include-install-pnpm.yml

```diff
@@ -24,7 +24,8 @@ steps:
 - ${{ if eq(parameters.enableCache, true) }}:
   - task: Cache@2
     displayName: Cache pnpm store
-    timeoutInMinutes: 5
+    timeoutInMinutes: 2
```

Comment:
First run I looked at (https://dev.azure.com/fluidframework/public/_build/results?buildId=159287&view=logs&j=eefa678c-f484-5837-78d3-627f1635913b&t=4303839d-0eab-5783-bad5-f520678e1acf ) took over two minutes for both cases. 2 might be a bit short.

Reply:
Updated to 3.